### PR TITLE
Change password

### PR DESCRIPTION
**Pull request by loemhuolay**
Change Password
- old password
- new password
- confirm password

For change password: <`http://localhost:3000/users/id/edit`>

### CODE REVIEW

File: pj_movie/config/routes.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-
+  get 'users/edit'
```
Comment (N310) [inline pj_movie/config/routes.rb]: you should move this line to under of root.

File: pj_movie/app/assets/stylesheets/users.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+.label {
+  display: flex;
+  flex-direction: column;
+  font-size: 18px;
+}
+input {
```
Comment (N310) [inline pj_movie/app/assets/stylesheets/users.scss]: you had better check in 320px width.

File: pj_movie/spec/controllers/users_controller_spec.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :controller do
```
Comment (N310) [inline pj_movie/spec/controllers/users_controller_spec.rb]: we should use request spec for controller test.

- Reference
https://relishapp.com/rspec/rspec-rails/docs/request-specs/request-spec

File: pj_movie/config/routes.rb [hunk 4]
```diff
@@ -1,4 +1,9 @@
 Rails.application.routes.draw do
+  resource :users do
+    collection do
+      get 'edit'
+    end
+  end
```
Comment (N310) [inline pj_movie/config/routes.rb]: you don't need to use `collection `.
you should move to under the root.

File: pj_movie/config/routes.rb [hunk 5]
```diff
@@ -2,8 +2,17 @@
   root 'static_pages#index'
   get 'admin', to: 'admin/top_page#index', as: :admin
 
+  resources :users, only: [:edit]
+
   namespace :admin do
-    resources :top_page, only: [:index]
+    resources :users, only: [:edit]
```
Comment (N310) [inline pj_movie/config/routes.rb]: I recommend you move this line to under the line 16.

File: pj_movie/config/routes.rb [hunk 6]
```diff
@@ -5,16 +5,10 @@
...
-    resources :users, only: [:new] do
-      collection do
-        get 'login'
-      end
-    end
-  end
-  resources :users, only: [:new] do
-    collection do
-      get 'login'
-    end
```
Comment (N310) [inline pj_movie/config/routes.rb]: Why do you remove routing  for singup and login?

File: pj_movie/db/schema.rb [hunk 7]
```diff
@@ -0,0 +1,56 @@
+ActiveRecord::Schema.define(version: 20191207022341) do
```
Comment (N310) [inline pj_movie/db/schema.rb]: you have to delete this file.

**Comment by R-Bear** [comment]
You should remove password reset from your change_password branch.

File: pj_movie/config/routes.rb [hunk 8]
```diff
@@ -11,12 +11,16 @@
         get 'login'
       end
     end
+    resources :purchases, only: [:show]
+    resources :top_page, only: [:index]
```
Comment (N310) [inline pj_movie/config/routes.rb]: You should remove this lines.

File: pj_movie/config/routes.rb [hunk 9]
```diff
@@ -11,12 +11,16 @@
...
+    resources :users, only: [:edit]
```
Comment (N310) [inline pj_movie/config/routes.rb]: You merge this line and 9th line.

File: pj_movie/config/routes.rb [hunk 10]
```diff
@@ -11,12 +11,16 @@
...
   end
   resources :users, only: [:new] do
     collection do
       get 'login'
     end
   end
+  resources :users, only: [:edit]
```
Comment (N310) [inline pj_movie/config/routes.rb]: You merge this line and 18th line.

**Pull request merged**